martyrs-devhub/gradeflow
# Updating and Fixing Files

# Description
This pull request aims at making sure the files are integrated with the new database changes, changing the UI/UX designs for the different entities of the platform that is the student, admin and also adding the technical team to the top navbar. Adding roles to the sql file to fit the admins and later, we aim at giving privileges to the admins basing on their roles.

## Type of change


## Trello Ticket ID
https://trello.com/c/9WzNrrjx

## How Can This Been Tested?

Fetch the remote branch and test it locally. Checkout the different platform entities and see that you can add new data, update , delete data basing on the new srms_version_1.sql file. But make sure you first pull the changes for the new sql file so that you are update.


## Checklist:


# Screenshots

![image](https://github.com/user-attachments/assets/61772a6e-9861-45fd-b877-0d4f846305a7)
